Index: src/Moduls/Door.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Moduls;\r\n\r\npublic class Door {\r\n\r\n    private boolean isOpen;\r\n\r\n    public boolean getIsOpen() {\r\n        return isOpen;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Moduls/Door.java b/src/Moduls/Door.java
--- a/src/Moduls/Door.java	(revision 1f88aaffc0500f40fb67bff072539ba6ea454233)
+++ b/src/Moduls/Door.java	(date 1667501155678)
@@ -1,9 +1,7 @@
 package Moduls;
 
 public class Door {
-
     private boolean isOpen;
-
     public boolean getIsOpen() {
         return isOpen;
     }
